support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -168,25 +169,3 @@ class CreateSubscriptionController(
   }
 
 }
-
-class LoggingCirceParser(controllerComponents: ControllerComponents) extends Circe {
```

Comment:
when I wrote this code I was just delighted that it actually worked, but having come back I have implemented it as an EssentialAction which is more appropriate, and combined with the AlarmOnError code.